components/automate-ui/package.json

```diff
@@ -81,6 +82,7 @@
     "@types/jasminewd2": "^2.0.8",
     "@types/lodash": "^4.14.168",
     "ajv": "^7.2.1",
+    "ansi-regex": "^5.0.1",
```

Comment:
Do we need to add in both dev and build dependencies? Same for chef-ui-library

Reply:
- According to Dependabot, when I was attempting to fix it, Dependabot suggested this solution. I also conducted some research on this, and I believe this is the correct way to resolve the issue. Here are the suggested changes from Dependabot.

![image](https://github.com/chef/automate/assets/12297653/a6cfc19b-1a71-4a3c-9f08-9d2bd824a9ac)

Reply:
It is only a suggestion to use based on requirement. Dependabot examples always shows in both dependency. We need to add only where it is required. The only point is the version should match what is suggested by dependabot.